pkg/agent/route/route_linux.go

```diff
@@ -737,10 +748,12 @@ func (c *Client) syncIPTables() error {
 	// for performance reasons.
 	addFilterRulesToChain(iptablesFilterRulesByChainV4, &c.wireguardIPTablesIPv4)
 	addFilterRulesToChain(iptablesFilterRulesByChainV4, &c.nodeNetworkPolicyIPTablesIPv4)
+	addFilterRulesToChain(iptablesFilterRulesByChainV4, &c.nodeLatencyMonitorIPTablesIPv4)
```

Comment:
check the comment above:

> Install the static rules (WireGuard for now) before the dynamic rules (e.g., NodeNetworkPolicy) for performance reasons

this applies to the NLM rules as well, which are also static